packages/cluster/src/defaults.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    dhis2Home: '/opt/dhis2',
```

Comment:
(minor) As far as I can tell this will never be used because it is always overridden by the "computed" dhis2home (common.js::168), is that correct and if so should this default be removed...?

Reply:
I think you are right 👌🏻 I thought I add it here to be consistent since others like `dhis2Version` also have a "default" even though for `dhis2Version` the default is `''`.

Keeping this default or removing is both fine for me. Which do you prefer @amcgee @radnov ?

Reply:
Given that `dhis2Version` is also in the defaults with an empty value, maybe we can either keep it as `/opt/dhis2` (if we assume this is the default home location for images going forward) or also keep it with an empty value. I'm also fine either way. 🤷‍♂️  😅